cabal-install/src/Distribution/Client/List.hs

```diff
@@ -466,8 +464,7 @@ showPackageDetailedInfo pkginfo =
         <<>> maybe Disp.empty (\v -> char '-' Disp.<> pretty v) (selectedVersion pkginfo)
       <+> text (replicate (16 - length (prettyShow (pkgName pkginfo))) ' ')
         <<>> parens pkgkind
-      $+$ ( nest 4 $
-              vcat
+      $+$ nest 4 (vcat
```

Comment:
Closing bracket 30+ lines later, odd but followable.

Reply:
I could have used indentation but didn't want to unindent so many lines, aiming for a small change.

Reply:
The closing `)`  was also far before right?

Reply:
Yes, +1 to Andrea's comment: it was far away in the initial version too.